AIFG-3D-Connect-4/TurnHandler.cpp

```diff
@@ -0,0 +1,44 @@
+#include "TurnHandler.h"
+
+TurnHandler::TurnHandler() : m_coordinate(0,0,0), m_playersTurn(true)
+{
+	// Set the players to nullptr, the user will set these up later.
+	m_player1 = nullptr;
```

Comment:
Probably best to do this in the initialiser list above like the coordinate and bool